packages/benchmark/src/config/config.ts

```diff
@@ -0,0 +1,50 @@
+import convict from 'convict';
```

Comment:
Same as with oclif - would we pick convict today? I've been moving env vars from `cli` to the new `@n8n/config` package, with a view to eventually removing `convict`. Any reason these new env vars cannot be in the new package?

Reply:
`convict` works well enough for this use case. Why do we want to remove it?

I wouldn't put these into `@n8n/config` as that package defines the config for `n8n`. This cli is its own executable with its own config.